src/smefit/cli/__init__.py

```diff
@@ -217,24 +217,18 @@ def report(report_card: pathlib.Path):
     type=float,
     default=None,
     required=False,
-    help="Adjusts the statistical uncertainties according to the specified luminosity",
+    help="Adjusts the statistical uncertainties according to the specified luminosity. If not specified, the original "
+    "uncertainties are kept and the central values are fluctuates according to the specified noise level.",
 )
 @click.option(
-    "--closure",
-    type=bool,
-    is_flag=True,
-    default=False,
-    help="Produces datasets under the SM",
+    "--noise",
+    type=str,
+    default="L0",
+    required=False,
+    help="Noise level for the projection, choose between L0 or L1. Assumes L0 by default.",
```

Comment:
```suggestion
    "--noise",
    type=click.Choice(["L0", "L1"], case_sensitive=True),
    default="L0",
    required=False,
    help="Noise level for the projection, choose between L0 or L1. Assumes L0 by default.",
```
This should enforce that only L1 and L0 are recognised otherwise gives error, would be good to test it.

Reply:
From the current implementation, I believe that if one gives --noise l1 with l lowercase, one would get L0 data.